### ISSUE TITLE
[CAIA A11y Research - PILOT]: VFF/Auth. Exp.: Confirmation page with Status, My VA (May EOM 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#83814]
> [!IMPORTANT]
> - **TICKET STATUS:** `Requested` Matt has requested a pilot for his project. 
> - **PILOT DATE:**  `5/29` due to holiday and research starting likely 6/3.
> - **TYPES OF ASSISTIVE TECH:** JAWS (Windows, using Edge)

# CAIA A11y Support
This is a child ticket of the research ticket [83382](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83382) - created to track work, completed from this pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- @coforma-jamie 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name:VFF and Authenticated Experience 
- OCTO-DE product owner: Matthew Dingee and Chante Lantos-Swett
- Product manager: Tricia Baker and Ana Jakabcin (AJ)
- Designer: Jeana Clark and Matt Marino
- FE engineers: Nick Sprinkle, Jacob Penner, Robert Hasselle, and Allison Lu
- Product/team Slack channel: #veteran-facing-forms and #accountexp-authexp

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** `TBD` 5/28, 5/29 or 5/30, due to holiday and research starting likely 6/3.
- [Research Folder with guides and plan](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/tree/main/Product/Form%20Status/Research) 
    - _Still being modified at the time of this posting by Matt in original ticket's comments 5/21/2024._
        - **Prototype**: `Prototype link when available and location CodePen, Staging, etc.`
        - **Research Convo Guide**: `Link when available`
        - **Research Plan**: `Link when available`

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Request CAIA a11y:** 5/21/2024 request by Matt; 5/22/2024 response for scheduling; waiting for team's availability. [See comments](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83382#issuecomment-2123426088).
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:**  [See pilot planning comment. ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83382#issuecomment-2125850473)
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled (Holds placed 5/22/2024)
- [ ] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Product/Form%20Status/Research/conversation-guide.md) and provide feedback in comments.
- [x] [Request AT device and tech to be used in sessions](LINK)
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot
- [ ] **Review prototype** in `CodePen` (linked at the top of the [conversation guide](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Product/Form%20Status/Research/conversation-guide.md)
- [ ] **Run pilot** with VFS Researcher
- [ ] **Optional: Note Taking:** If there is only one a11y, notes cannot be taken due to pilot participation. Folder for Team in GH Deliverables for CAIA A11Ys.
```

**Comment by coforma-jamie** [comment]
@mattmarino-adhoc, I took a look at your conversation guide. On the whole, it's great! I just have a couple of suggestions:

- In the beginning, you ask the participant what kind of "computer" they're on. If you're expecting mobile and tablet users to be part of the study as well, you might want to replace that with "device" or something similar.
- A couple of things about links to the different prototypes. Since there are a bunch of links you'll be sharing, and some users may end up wanting links dictated letter by letter, I recommend:
   - Creating bit.ly links for all of them, for ease of dictating
   - Having an accessible email of _all_ the links on standby, that you can send to them if they want links via email. The email could be structured like...
   ~~~
   - [link] Prototype 1
   - [link] Prototype 2
   - [link] Prototype 3
   - etc
   ~~~
   Then you can simply tell them, "go back to the email we sent you, and pull up Prototype 3."

**Comment by coforma-jamie** [comment]
The pilot is complete. The team took notes and will get back to me to review changes to the prototypes later on this week.
